lib/formtools/overview.js

```diff
@@ -0,0 +1,183 @@
+
+var async = require('async'),
+    cellRenderers = require('./renderers-cell');
+
+/**
+ * Set unique tabId to each tab in overview.body
+ * @param {Array} overviewBody overview body
+ */
+function setTabId (overviewBody) {
+
+    overviewBody.forEach(function (section, sectionIndex) {
+
+        if (Array.isArray(section)) {
+
+            section.forEach(function (tab, tabIndex) {
+
+                // Replace white space from tab.lable with "-", make it lowecase and add the index number at the end.
+                // sectionIndex and tabIndex number are added at the end to make the tabID unique in case when more then one tab have the same label (title)
+                tab.tabId = (tab.label).replace(/\s+/g, '-').toLowerCase() + '-' + sectionIndex + '-' + tabIndex;
+
+            });
+        }
+    });
+
+}
+
+/*
+ * Replace each field name in section of overview.body with object containing field label and filed value
```

Comment:
Perhaps update this description to `Update section.fields to an array of objects with label and value keys. Label should contain the field label and value should contain the field value.`